js/tests/unit/dropdown.spec.js

```diff
@@ -1358,7 +1358,7 @@ describe('Dropdown', () => {
         btnDropdown.addEventListener('shown.bs.dropdown', () => {
           expect(btnDropdown).toHaveClass('show')
 
-          const keyup = createEvent('keyup')
+          const keyup = createEvent('keyup', { bubbles: true })
```

Comment:
This and all other `createEvent` calls I changed are for events that bubble in the browser when they come from keyboard/mouse input. It might be reasonable to change the default for `createEvent`? I'm not sure. I only changed as many calls as were required to get the tests passing.